src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,543 @@
+                if (isPseudoAbbreviation != null) {
+                    if (!isEmpty) {
+                        if (isEditableAndRemovable.get()) {
+                            if (!isPseudoAbbreviation) {
+                                Text graphic = new Text(IconTheme.JabRefIcon.EDIT.getCode());
+                                graphic.getStyleClass().add("icon");
+                                setGraphic(graphic);
+                                setOnMouseClicked(evt -> {
+                                    editAbbreviation();
+                                });
+                            } else {
+                                Text graphic = new Text(IconTheme.JabRefIcon.ADD.getCode());
+                                graphic.getStyleClass().add("icon");
+                                setGraphic(graphic);
+                                setOnMouseClicked(evt -> {
+                                    addAbbreviation();
+                                });
+                            }
+                        }
+                    }
+                } else {
+                    setGraphic(null);
+                }
+            }
+        });
+        journalTableDeleteColumn.setCellFactory(column -> new TableCell<AbbreviationViewModel, Boolean>() {
+
+            @Override
+            protected void updateItem(Boolean isPseudoAbbreviation, boolean isEmpty) {
+                super.updateItem(isPseudoAbbreviation, isEmpty);
+                if (isPseudoAbbreviation != null) {
+                    if (!isEmpty && isEditableAndRemovable.get()) {
+                        if (!isPseudoAbbreviation) {
+                            Text graphic = new Text(IconTheme.JabRefIcon.DELETE_ENTRY.getCode());
+                            graphic.getStyleClass().add("icon");
+                            setGraphic(graphic);
+                            setOnMouseClicked(evt -> {
+                                removeAbbreviation();
+                            });
+                        } else {
+                            setGraphic(null);
+                        }
+                    }
+                } else {
+                    setGraphic(null);
+                }
+            }
+
+        });
+    }
+
+    private void setBindings() {
+        viewModel.currentFileProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalFilesBox.getSelectionModel().select(newvalue);
+            if (newvalue != null) {
+                isEditableAndRemovable.set(!newvalue.isPseudoFileProperty().get());
+            }
+            removeJournalAbbreviationsButton.setDisable((newvalue == null) || !isEditableAndRemovable.get());
+        });
+        viewModel.currentAbbreviationProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalAbbreviationsTable.getSelectionModel().select(newvalue);
+        });
+        journalAbbreviationsTable.getSelectionModel().selectedItemProperty()
```

Comment:
`journalAbbreviationsTable.getSelectionModel().selectedItemProperty()` is read only and can therefore not be bound. The `journalAbbreviationsTable.itemsProperty()` is a `ObjectProperty<ObservableList<AbbreviationViewModel>>` which unfortunately can not be bound to my `SimpleListProperty<AbbreviationsFileViewModel>.